### PR TITLE
Makefile setup.

### PR DESCRIPTION
**Pull request by perusio**
+ Implements Makefile for building, pushing, running.
+ Improved Dockerfile.
+ README updated.
+ Fixed modes of Bash scripts.
+ Included missing pytest modules: mypy and cov.

### CODE REVIEW

File: blocks/land_cover_classification/requirements.txt [hunk 1]
```diff
@@ -1,13 +1,63 @@
-numpy==1.16.2
```
Comment (markusuwe) [inline blocks/land_cover_classification/requirements.txt]: Why all these additional requirements?

**Comment by markusuwe** [comment]
Thanks for the fix of the requirements.

File: README.md [hunk 2]
```diff
@@ -12,139 +19,242 @@ geospatial pipeline. It shows how easy it is to setup a clustering
...
-First create a virtual environment either by using [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/) 
-or [virtualenv](https://virtualenv.pypa.io/en/latest/).
-In the case of using virtualenvwrapper do:
+where `<directory>` is the directory where the cloning is done.
 
-```mkvirtualenv --python=$(which python3.7) up42-landcover```
+### Build the docker image
 
-In the case of using virtualenv do:
+For building the image you should tag the image such that it can bu
```
[the remark this answers is not in the document]
Comment (nekhtiari) [reply README.md]: Other than this, LGTM

[the remark this answers is not in the document]
Comment (perusio) [reply README.md]: @nekhtiari fixed thanks

**Pull request merged**